### PR TITLE
Remove crc specific parameters from kuttl tests

### PR DESCRIPTION
**Pull request by cescgina**
Remove some fields from the asserts (host in routes and apiEndpoints in
CR spec) that had crc specific values since those make the tests fail if
run in a cluster other than crc. It should be easy to check with
a regex, but kuttl does not support it. If they're really important to
check, can introduce a TestAssert for them and check the value with some
script.

### CODE REVIEW

**Review by stuggi — APPROVED** [review]
/lgtm

File: tests/kuttl/common/assert_sample_deployment.yaml [hunk 1]
```diff
@@ -43,10 +43,6 @@ spec:
       memory: 500Mi
   secret: osp-secret
 status:
-  apiEndpoint:
-    admin: http://keystone-admin-openstack.apps-crc.testing
-    internal: http://keystone-internal-openstack.apps-crc.testing
-    public: http://keystone-public-openstack.apps-crc.testing
```
Comment (gibizer) [inline tests/kuttl/common/assert_sample_deployment.yaml:49]: Unfortunately asserting that keystone has endpoints is part of the functionality we want to check here. So while I understand that the data here is deployment specific (ie. different in CRC than in SNO) simply not asserting such data feels wrong to me too. Unfortunately I cannot offer a solution here. Kuttl's assert possibilities seems limiting here.

Comment (stuggi) [reply tests/kuttl/common/assert_sample_deployment.yaml:49]: like run a command which uses a regexp?

Comment (cescgina) [reply tests/kuttl/common/assert_sample_deployment.yaml:49]: We could check with some script, it'll be less clean, but should be doable. Let me try and report back

Comment (gibizer) [reply tests/kuttl/common/assert_sample_deployment.yaml:49]: Yeah having a check that we have all the endpoints with some URL but ignoring the actual value of the URL would be better. I don't like having all these moving towards bash scripts as asserts but I don't see another option right now.

**Comment by cescgina** [comment]
@stuggi @gibizer I've added a script to validate the endpoints urls. It assumes that the first part is always set to the same string (seems to be true for both CRC and SNO).

**Review by gibizer — APPROVED** [review]
Thanks

**Review by stuggi — APPROVED** [review]
/lgmt thanks!

**Pull request merged**